src/action/info.js

```diff
@@ -38,6 +38,7 @@ class InfoAction {
         });
         this._store.percentSynced = this.calcPercentSynced(response);
       } else {
+        this._store.wallet.restoring = false;
```

Comment:
When is `store.wallet.restoring` set to true?

Reply:
On the `select-seed` view, when the user selects that they're restoring a wallet.

Reply:
Yes, sorry thought i fixed that